capabilities/src/execute_task_solution_capability.cpp

```diff
@@ -94,8 +94,10 @@ void ExecuteTaskSolutionCapability::initialize() {
 	                                             std::placeholders::_1, std::placeholders::_2)),
 	    ActionServerType::CancelCallback(
 	        std::bind(&ExecuteTaskSolutionCapability::preemptCallback, this, std::placeholders::_1)),
```

Comment:
For consistency, can you convert the goal and cancel callbacks to use lambdas instead of `std::bind` as well?